src/module/item/data-model-ability.js

```diff
@@ -27,9 +29,11 @@ export default class OseDataModelAbility extends foundry.abstract.DataModel {
         : ` ${CONFIG.OSE.roll_type[this.rollType]}${this.rollTarget}`;
```

Comment:
Guessing we can't move this stuff into the helper, too?

No biggie if not. A little tidier is still tidier.

Reply:
I added a new helper named rollTagTarget to generate those as well and removed them from spells since it is not included with spell data anyway.